emscripten.py

```diff
@@ -29,14 +29,6 @@ def path_from_root(*pathelems):
   """
   return os.path.join(__rootpath__, *pathelems)
 
-def get_configuration():
-  if hasattr(get_configuration, 'configuration'):
```

Comment:
wat

Reply:
yeah, i never really understood this part of the code myself